### DISCUSSION TITLE
Difference between different deployment URLs

### DISCUSSION THREAD

**Discussion by jason-roundtree** [vercel/vercel#4546] [Help] [answered]
I'm a new user of Next.js and I just deployed my first Next.js app through Vercel using Github integration and every time I push I receive an email from notifications@github.com with 4 different Vercel URLs in the following format: 
1. `https://<github-repo>-<some-id>.now.sh`
2. `https://<github-repo>-rho-one.now.sh`
3. `https://<github-repo>-<github-user-handle>.now.sh`
4. `https://<github-repo>-git-master.<my-github-user-handle>.now.sh`

When I look at the Overview tab of my Vercel dashboard I see two of these URLs: URL number 2 above says 'Production' next to it; URL number 1 above says 'Latest'. I assume Production is what I should publish to the world. Is Latest used to revert to older deployments? What are the differences between each of these URLs? Thanks and apologies if this isn't the correct place to ask this!

**Comment by markozxuu** [comment, marked as answer]
I will try to explain it, I hope my answer can be of help to you 🙏 

https://markozxuu-[hash].now.sh: this url is generated every time a deployment is triggered, the advantage of this is that you always have a history of the changes your project went through, think of this as a git commit, each commit has a different hash : )

https://markozxuu.[username_vercel].now.sh: this url is usually the one that is shared to anyone who wants to see your project online.

https://markozxuu.now.sh: this is the production url for your site. This is the one you will use in the real world, as you can see it does not have the username that you have in your Vercel account

https://markozxuu-git-master.justinmark.now.sh: url generated by GitHub, the latter is not very important

<img width="468" alt="Screen Shot 2020-06-06 at 22 26 14" src="https://user-images.githubusercontent.com/16585386/83959626-bd932100-a844-11ea-8fb2-8e13caffe7a2.png">

**Reply by jason-roundtree** [reply]
Thank you!
